flocker/provision/_rackspace.py

```diff
@@ -15,6 +16,14 @@ def provision_rackspace(node, package_source, distribution):
     """
     Provision flocker on this node.
     """
+    if distribution in ('centos-7',):
+        run(
+            username='root',
+            address=node.address,
+            commands=task_upgrade_kernel_centos(),
```

Comment:
Now that I've digested more of this, I realize the docs that say to upgrade the kernel are meant to support any random CentOS 7 machine.  Since we don't know what kernel such a machine will have, a blanket statement to upgrade seems alright.  And then this code is specifically to support Rackspace, and maybe we know that that system in particular does need a kernel upgrade, and we also know that the AWS and DigitalOcean CentOS 7 images don't need the upgrade so we can skip this for them.

Is that right?